demo_umami_content/src/InstallHelper.php

```diff
@@ -244,6 +245,51 @@ public function importPages() {
     return $this;
   }
 
+  /**
+   * Imports block contents.
+   *
+   * @return $this
+   */
+  public function importBlockContent() {
+    $block_content_metadata = [
+      'umami_recipes_banner' => [
+        'uuid' => '4c7d58a3-a45d-412d-9068-259c57e40541',
+        'info' => 'Umami Recipes Banner',
+        'type' => 'banner_block',
+        'fields' => [
+          'field_title' => [
+            'value' => 'Baked Camembert with garlic, calvados and salami',
+          ],
+          'field_content_link' => [
+            'target_id' => 10,
+          ],
+          'field_summary' => [
+            'value' => 'Nullam id dolor id nibh ultricies vehicula ut id elit. Nullam id dolor id nibh ultricies vehicula ut id elit. Nullam id dolor id nibh ultricies vehicula ut id elit.',
+          ],
+          'field_banner_image' => 'images/jpg/placeholder--atharva-lele-210748-pshopped.jpg',
+        ],
+      ],
+    ];
+
+    // Create block content.
+    foreach ($block_content_metadata as $block) {
+      $block_content = $this->entityTypeManager->getStorage('block_content')
```

Comment:
if you remove the 'fields' key, you should be able to `->create($block)` and then only need to set the image value.